apps/gitea-mirror/overlays/nishir-tailnet/kustomization.yaml

```
@@ -21,3 +23,10 @@ secretGenerator:
   - name: gitea-mirror
     envs:
       - gitea-mirror/.enc.env
+patches:
```

YAMLlint (1.37.1)

[error] duplication of key "patches" in mapping

(key-duplicates)